### DISCUSSION TITLE
Problem with websocket

### DISCUSSION THREAD

**Discussion by 1Mack** [vercel/community#3841] [Help]
### Summary

I've configured my page to work with websesocket and it was working properly on my preview environment, but when i changed it to the production, it stopped working. Im literally using the same build for both environments.

Does anyone know what should i do to fix it?

Obs: Both domains are correctly configured.

### Steps to Reproduce

WebSocket connection to 'wss://*********/ws' failed:

**Comment by amyegan** [comment]
Hey @1Mack. If you share your public repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will give us a better look at what is going wrong.
